assets/js/services/helpers.js

```diff
@@ -166,3 +166,10 @@ export function formatBytes(bytes, decimals) {
     i = Math.floor(Math.log(bytes) / Math.log(k));
   return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + " " + sizes[i];
 }
+
+export function truncateText(text, number) {
+  if (text.length <= number) {
+    return text;
+  }
+  return text.slice(0, number) + "...";
+}
```

Comment:
This totally works, but wouldn't hurt to provide a default param to `text` (`truncateText(text = "", number)`).

And if you wanted to learn something cool about forward-thinking React code, this helper (and all helpers for that matter) could be a hook instead.   In 2 years, say someone other than us was reading the code the convention may be more familiar to them.

```
export default function useTruncateText() {
  function truncate(text, number) {
  ... 
  }
  return { truncate };
}

// usage
import useTruncateText from '@js/hooks/use-truncate-text'
...
const { truncate } = useTruncateText();
truncate("foobaz", 50);
```